tox.ini

```diff
@@ -0,0 +1,28 @@
+[tox]
+envlist = py26,py27,lint,docs
+skipsdist = True
+
+[testenv]
+setenv =
+    PYTHONPATH={toxinidir}/readthedocs:{toxinidir}
+    DJANGO_SETTINGS_MODULE=settings.test
+deps = -r{toxinidir}/requirements/pip.txt
+changedir = {toxinidir}/readthedocs
+commands =
+    py.test {posargs}
```

Comment:
in `runtests.sh` we use `LANG=C coverage run -m pytest`. The reason for the LANG=C is that if you don't have a english locale set in the system, the tests will fail with this assertion:

```
E       AssertionError: Regexp didn't match: '(?:No such file or directory|not found)' not found in '/bin/sh: /non-existant/foobar: Datei oder Verzeichnis nicht gefunden\n'
```

We should use `LANG=C` with tox as well.